lib/block_view.go

```diff
@@ -6624,6 +6818,22 @@ func (bav *UtxoView) _connectSwapIdentity(
 	bav._setPKIDMappings(&newFromPKIDEntry)
 	bav._setPKIDMappings(&newToPKIDEntry)
 
+	// Postgres doesn't have a concept of PKID Mappings. Instead, we need to save an empty
+	// profile with the correct PKID and public key
+	if bav.Postgres != nil {
```

Comment:
Hmm... why change the semantics here vs having a PKID<->public_key table?

Reply:
There's no need to store PKID<->Public Key until a swap identity occurs or a profile is created. It's much more space efficient this way.